menpo/io/input/base.py

```diff
@@ -44,13 +46,13 @@ def data_path_to(asset_filename):
     return asset_path
 
 
-def same_name(asset):
+def same_name(path):
     r"""
-    Menpo's default landmark resolver. Returns all landmarks found to have
+    Menpo's default image landmark resolver. Returns all landmarks found to have
     the same stem as the asset.
     """
     # pattern finding all landmarks with the same stem
-    pattern = asset.path.with_suffix('.*')
```

Comment:
Don't mind this change, but it is another breaking one, can you note it in the PR notes so we catch it in the changelog?